src/topology/algebra/ordered.lean

```diff
@@ -394,32 +416,32 @@ begin
       exact λ c hc, lt_of_lt_of_le (hf.2 hc) xb } },
 end
 
-theorem induced_orderable_topology {α : Type u} {β : Type v}
-  [partial_order α] [ta : topological_space β] [partial_order β] [orderable_topology β]
+theorem induced_order_topology {α : Type u} {β : Type v}
+  [partial_order α] [ta : topological_space β] [partial_order β] [order_topology β]
   (f : α → β) (hf : ∀ {x y}, f x < f y ↔ x < y)
   (H : ∀ {x y}, x < y → ∃ a, x < f a ∧ f a < y) :
-  @orderable_topology _ (induced f ta) _ :=
-induced_orderable_topology' f @hf
+  @order_topology _ (induced f ta) _ :=
+induced_order_topology' f @hf
   (λ a x xa, let ⟨b, xb, ba⟩ := H xa in ⟨b, hf.1 ba, le_of_lt xb⟩)
   (λ a x ax, let ⟨b, ab, bx⟩ := H ax in ⟨b, hf.1 ab, le_of_lt bx⟩)
 
-lemma nhds_top_orderable [topological_space α] [order_top α] [orderable_topology α] :
+lemma nhds_top_order [topological_space α] [order_top α] [order_topology α] :
   𝓝 (⊤:α) = (⨅l (h₂ : l < ⊤), principal (Ioi l)) :=
-by simp [nhds_eq_orderable (⊤:α)]
+by simp [nhds_eq_order (⊤:α)]
 
-lemma nhds_bot_orderable [topological_space α] [order_bot α] [orderable_topology α] :
+lemma nhds_bot_order [topological_space α] [order_bot α] [order_topology α] :
   𝓝 (⊥:α) = (⨅l (h₂ : ⊥ < l), principal (Iio l)) :=
-by simp [nhds_eq_orderable (⊥:α)]
+by simp [nhds_eq_order (⊥:α)]
 
 section linear_order
 
-variables [topological_space α] [linear_order α] [t : orderable_topology α]
+variables [topological_space α] [linear_order α] [t : order_topology α]
```

Comment:
Why is this type class naming necessary? This is usually the sign that a lemma is missing.

Reply:
This wasn't necessary. I have checked that the other named type classes in this file correspond to the beginning of the theory, when one wants to restate the definition of the type class is one way or another.